test/muleCI/mule.yaml

```diff
@@ -215,33 +216,32 @@ jobs:
     tasks:
       - docker.Make.build.amd64
       - stash.Stash.linux-amd64
-  build-linux-arm64:
-    tasks:
-      - docker.Make.build.arm64
-      - stash.Stash.linux-arm64
-  build-linux-arm32:
-    tasks:
-      - docker.Make.build.arm
-      - stash.Stash.linux-arm
+#  build-linux-arm64:
```

Comment:
I think a bunch of lines were left commented out in this file.

Reply:
Uncommented.